packages/api/tests/endpoints/collections/delete-collection.js

```diff
@@ -493,3 +495,37 @@ test.serial('del() does not remove from PostgreSQL or publish SNS message if rem
 
   t.is(Messages, undefined);
 });
+
+test.serial('del() deletes a collection and removes its configuration store via name', async (t) => {
+  const newCollection = fakeCollectionFactory();
+
+  await request(app)
+    .post('/collections')
+    .send(newCollection)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  t.true(await t.context.collectionPgModel.exists(t.context.testKnex, {
+    name: newCollection.name,
+    version: newCollection.version,
+  }));
+  t.truthy(await collectionConfigStore().get(newCollection.name, newCollection.version));
```

Comment:
```suggestion
  t.like(await collectionConfigStore().get(newCollection.name, newCollection.version), newCollection);
```

If possible, it's preferable to do a stricter test assertion because `truthy` can pass for lots of different values